plugins/CoreHome/Tracker/VisitRequestProcessor.php

```diff
@@ -112,6 +112,10 @@ public function processRequestParams(VisitProperties $visitProperties, Request $
         $isNewVisit = $this->isVisitNew($visitProperties, $request);
         $request->setMetadata('CoreHome', 'isNewVisit', $isNewVisit);
 
+        if (!$isNewVisit) { // only copy over known visitor's information, if this is for an ongoing visit
+            $this->visitorRecognizer->updateVisitPropertiesFromLastVisitRow($visitProperties);
```

Comment:
@diosmosis just curious was there a specific problem re moving it out here? Also wondering if we know for sure whether `CoreHome/RequestProcessor` will be executed first or whether other request processors may expect it to be set already? Is it because we might now find more often a `visitRow` but need to create a new visit because it is older than 30minutes maybe?

Reply:
We don't know it's a new visit until this point. Before this point we only know if it's a known visitor (the last action might be from a visit that is > 30 mins). Since we weren't looking further than that before, it wasn't an issue, but now it's possible to find a log from a different visit.

Reply:
Can we somehow make sure that `CoreHome` request processor will be executed first?

Reply:
Just to prevent any regressions

Reply:
Can look into that, though this used to be done in `findKnownVisitor` which is executed a couple lines above, so it's not like this code was moved from core => the CoreHome plugin. Ie, this request processor is executed at the same time it was before this change.

Reply:
got it. didn't realise that. shouldn't be an issue then 👍